src/test/java/edu/oakland/test/display01/DisplayGpsReceiverStub.java

```diff
@@ -0,0 +1,34 @@
+package edu.oakland.test.display01;
+
+import edu.oakland.helper.admin.Satellite;
+import edu.oakland.helper.display01.SatelliteSignalCheckRequest;
+import edu.oakland.production.display01.DisplayGpsReceiver;
+
+public class DisplayGpsReceiverStub implements DisplayGpsReceiver {
+
+  /** 
+   * Measures GPS Signal Strength.
+   *
+   * @param satelliteSignal GPS signal to measure.
+   * @return Status of GPS signal measured.
+   */
+  public String measureGpsSignalStrength(Satellite satelliteSignal) {
+    Satellite s = satelliteSignal;
+    return s.getSatelliteName();
+
```

Comment:
We can streamline this to:
```suggestion
    return satelliteSignal.getSatelliteName();
```